hackathon_site/hardware/tests.py

```diff
@@ -1,3 +1,71 @@
 from django.test import TestCase
 
-# Create your tests here.
+from hardware.models import Hardware, Category
+from hardware.serializers import HardwareSerializer, CategorySerializer
+
+
+class HardwareSerializerTestCase(TestCase):
+    def setUp(self):
+        self.hardware = Hardware.objects.create(
+            name="name",
+            model_number="model",
+            manufacturer="manufacturer",
+            datasheet="/datasheet/location/",
+            quantity_available=4,
+            max_per_team=1,
+            picture="/picture/location",
+        )
+
+    def test_hardware_serializer(self):
+        hardware_serializer = HardwareSerializer(self.hardware)
+        expected_response = {
+            "id": 1,
+            "name": "name",
+            "categories": [],
+            "model_number": "model",
+            "manufacturer": "manufacturer",
+            "datasheet": "/datasheet/location/",
+            "quantity_available": 4,
+            "quantity_remaining": 4,
+            "notes": None,
+            "max_per_team": 1,
+            "picture": "picture/location",
+        }
+
+        data = hardware_serializer.data
+        self.assertCountEqual(
```

Comment:
This is redundant if you're asserting equality between `data` and `expected_response`